src/reducers/Grid.js

```diff
@@ -20,8 +20,25 @@ const updateBlock = (state, { row, column, machineType }) => ({
   ],
 })
 
+const deletePosition = (gridValues, { row, column }) => {
+  const oldGridValues = gridValues
```

Comment:
why you are copy the array ?

Reply:
In my first try I tried to use the `lodash` copy but I forgot that we are not using that. So I didn't change that logic.
I'm copying that data because data should be immutable(as far as I know it is still a good practice).